gemrb/core/InterfaceConfig.h

```diff
@@ -23,14 +23,88 @@
 
 #include <string>
 #include <unordered_map>
+#include <vector>
 
 #include "exports.h"
+#include "System/VFS.h"
+
+#define VERSION_GEMRB "0.9.2-git"
```

Comment:
CMakeLists.txt and .appveyor.yml also need to be updated for this file move.

Reply:
I'll add a comment too then